spec/std/crystal/compiler_rt/divti3_spec.cr

```diff
@@ -0,0 +1,28 @@
+{% skip_file unless flag?(:bits32) %}
```

Comment:
Shouldn't this `unless` be `if`?
ditto elsewhere

Reply:
good catch

Reply:
I am on a mac that is 64bit and this flag is working as is.  When I change it to if 32bits it fails.

Reply:
I would expect that the specs can run in all the platforms. They call explicitly the functions.